internal-functions

```diff
@@ -122,5 +122,12 @@ fn-check-service-acl() {
 
 fn-query-admin-status(){
   HEADER="X-Plotly-Dash-Secret-Key: $1"
-  curl -sk --header "$HEADER" "https://$2/v2/users/$3/" | jq '.is_admin' | grep -q 'true'
+  AUTH_URL="$2"
+
+  if [[ "$PLOTLY_STREAMBED_AUTH_ONLY" == 1 ]]; then
+    AUTH_URL="$AUTH_URL/Auth"
+  fi
+
+  USER_INFO_URL="https://$AUTH_URL/v2/users/$3/"
+  curl -sk --header "$HEADER" "$USER_INFO_URL" | jq '.is_admin' | grep -q 'true'
```

Comment:
We should add a timeout here for connection and response.

Reply:
From https://unix.stackexchange.com/questions/94604/does-curl-have-a-timeout/94612#94612, it seems that I want to use the `--max-time` flag. Something like 60s seems reasonable to me. Any objections @josegonzalez?

Reply:
That seems like a very long time for us to wait for the auth server to respond. Maybe 3 seconds is better?

How long does the call normally take? Can we time that and multiply by 2 for safety?

Reply:
>How long does the call normally take? Can we time that and multiply by 2 for safety?

Both my local and GC test instances run that curl in less than a tenth of a second: 

```root@d56ea0d9c3de:/app# declare HEADER="X-Plotly-Dash-Secret-Key: $PLOTLY_DASH_SECRET_KEY"
root@d56ea0d9c3de:/app# declare AUTH_URL="$PLOTLY_STREAMBED_HOST/Auth"
root@d56ea0d9c3de:/app# declare USER_INFO_URL="https://$AUTH_URL/v2/users/admin/"
root@d56ea0d9c3de:/app# curl -sk -w "@curl-format.txt" --header "$HEADER" "$USER_INFO_URL" 
    time_namelookup:  0.000050
       time_connect:  0.000251
    time_appconnect:  0.046500
   time_pretransfer:  0.046577
      time_redirect:  0.000000
 time_starttransfer:  0.064608
                    ----------
         time_total:  0.064637
```

Setting it to 3 seconds seems reasonable to me!